stylus/ui-app/forms.styl

```diff
@@ -5,9 +5,158 @@
  * This file contains all needs for forms, inputs, labels...
  */
 
+@import '../ui-base/icons'
+
 checkbox-size = 20px
 
-$checkbox
+$form
+    .coz-form
+        position relative
+        margin   1em 0 0.5em
+
+        // Managing input width with flex
+        display         flex
+        flex-direction  column
+
+    label
+        display        block
+        text-transform uppercase
+        color          grey-11
+        font-size      .9em
+        padding        .55em 0
+        margin-top     1em
+
+    .coz-error-label
+        display none
+        color   red
+
+    .coz-errors
+        color red
+
+    a, a:visited
+        color               dodger-blue
+        text-decoration     none
+
+
+$form-button
+    .coz-form-controls
+        width      70%
+        padding     0 15%
+        display    flex
+        margin-top 1.5em
+
+        button
+            width       100%
+            margin      0.4em
+            cursor      pointer
+
+    [role=button],
+    button
+        // mutualize behavior between button and a
+        box-sizing        border-box
+        border            0
+        display           block
+        padding           0.9em 0.75em
+        min-height        (40/basefont)em
```

Comment:
You can use the `em(40)` function to direclty express a size in EM, relative to its context